changelog/unreleased/kong/fix-ai-plugin-no-consumer.yml

```diff
@@ -0,0 +1,4 @@
+message: "Fix certain AI plugins cannot be applied per consumer or per service."
```

Comment:
message: "Fixed certain AI plugins cannot be applied per consumer or per service."

Reply:
fixed